src/se/content.rs

```diff
@@ -133,7 +133,7 @@ impl<'i, W: Write> Serializer for ContentSerializer<'i, W> {
     type SerializeTupleStruct = Self;
     type SerializeTupleVariant = ElementSerializer<'i, W>;
     type SerializeMap = Impossible<Self::Ok, Self::Error>;
-    type SerializeStruct = Impossible<Self::Ok, Self::Error>;
+    type SerializeStruct = Struct<'i, W>;
```

Comment:
This is the wrong change. Content serializer represents data in a text, or, in your case, a binary, form. Structs and map a complex data objects with internal structure and cannot be serialized in such unstructured manner directly. If you want that, you should use `#[serde(serialize_with)]` attribute and convert your types to text / binary data manually

Reply:
That makes sense, thanks!